From 29c4721b18ad422cc3cbb6aed9d321900dab0e51 Mon Sep 17 00:00:00 2001
From: =?utf8?q?Martin=20Storsj=C3=B6?= <martin@martin.st>
Date: Thu, 1 Mar 2018 22:20:45 +0200
Subject: [PATCH] arm_neon: Include asm.S in distribution tarballs

Signed-off-by: Thomas Guillem <thomas@gllm.fr>
(cherry picked from commit 0d3878ef7488bab8bcb25b0f585143fea96e468f)
Signed-off-by: Thomas Guillem <thomas@gllm.fr>

Patch-Source: https://git.videolan.org/?p=vlc/vlc-3.0.git;a=commit;h=29c4721b18ad422cc3cbb6aed9d321900dab0e51
---
 modules/arm_neon/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules/arm_neon/Makefile.am b/modules/arm_neon/Makefile.am
index db81ec0b84..10f5c15d7c 100644
--- a/modules/arm_neon/Makefile.am
+++ b/modules/arm_neon/Makefile.am
@@ -28,3 +28,5 @@ neon_LTLIBRARIES = \
 	libvolume_neon_plugin.la \
 	libyuv_rgb_neon_plugin.la
 endif
+
+EXTRA_DIST += arm_neon/asm.S
-- 
2.11.0
